recipes/awscli2/meta.yaml

```diff
@@ -0,0 +1,143 @@
+{% set name = "awscli" %}
```

Comment:
```suggestion
{% set name = "awscli2" %}
```

Reply:
No, the package is still called `awscli` and can be differentiated by the version number.